appserver/admin/gf_template_web/src/main/resources/config/domain.xml

```diff
@@ -395,7 +396,7 @@
              </message-security-config>
          </security-service>
          <transaction-service tx-log-dir="${com.sun.aas.instanceRoot}/logs" automatic-recovery="true" />
-         <hazelcast-runtime-configuration></hazelcast-runtime-configuration>
+            <hazelcast-config-specific-configuration></hazelcast-config-specific-configuration>
```

Comment:
Why `hazelcast-runtime-configuration` element renamed to `hazelcast-config-specific-configuration` ?

Reply:
The `hazelcast-runtime-configuration` element is used to configure Domain wide Hazelcast Settings, and `hazelcast-config-specific-configuration` is only used to configure specific config.